cloud/config/config.go

```diff
@@ -68,7 +68,12 @@ func getConfigFromFile(path string) (*AuthConfig, error) {
 		return nil, err
 	}
 
-	defer f.Close()
+	deferCloseFunc := func() error {
```

Comment:
The wiki talks about writable files -> https://www.joeshaw.org/dont-defer-close-on-writable-files/ , in our case, the file is not writable, so this is not applicable to us right? We never call write() method.